temoto_core/Error[] error_stack
